### PR TITLE
Fix/credit card update menu

### LINKED ISSUE #455: Credit card choice menu
In **mitt-konto**, the user has the possibility to replace a credit card and update the corresponding subscriptions tied to it with a new one.

If the user has more than one credit card, they should be able to choose the one they want to replace. Right now we just take the first one we fetch from **bottega**.

Related to #412.

### PR DESCRIPTION
**Pull request by ae-mo**
Adds choice menu to credit card update functionality.

Closes #455.

### CODE REVIEW

File: packages/components/src/CreditCard/Menu.purs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import Bottega.Models (CreditCard)
+import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import KSF.CreditCard.Menu.Item (item) as CreditCard
+import KSF.Grid as Grid
+import React.Basic as React
+import React.Basic (JSX, make)
+import React.Basic.DOM as DOM
+import React.Basic.DOM.Events (preventDefault)
+import React.Basic.Events (handler)
```
Comment (kaol) [inline packages/components/src/CreditCard/Menu.purs:14]: Unused imports: `KSF.Grid`, `React.Basic.DOM.Events` and `React.Basic.Events`.

File: packages/components/src/CreditCard/Choice.purs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      { className: "credit-card-choice--description"
+      , children: [ DOM.text "Välj ett av dina kort ur listan nedan:" ]
+      } 
+
+    submitFormButton :: JSX
+    submitFormButton =
+      Grid.columnThird $
+        DOM.button
+          { type: "submit"
+          , children: [ DOM.text "Continue" ]
```
Comment (kaol) [inline packages/components/src/CreditCard/Choice.purs:96]: Shouldn't that be "Fortsätta"? (Or something, check with someone).

Comment (ae-mo) [reply packages/components/src/CreditCard/Choice.purs:96]: Aah good catch, totally forgot about it!

File: packages/components/src/CreditCard/Register.purs [hunk 3]
```diff
@@ -0,0 +1,44 @@
+module KSF.CreditCard.Register where
+
+import KSF.User (PaymentTerminalUrl)
+import React.Basic as React
+import React.Basic (JSX, make)
+import React.Basic.DOM as DOM
+
+type Props = 
+  { title :: JSX
+  , terminalUrl :: PaymentTerminalUrl }
```
Comment (kaol) [inline packages/components/src/CreditCard/Register.purs:10]: Styling: Looks like our convention has been to put the closing bracket on multi line data definitions on its own line.

File: packages/components/src/CreditCard/Update.purs [hunk 4]
```diff
@@ -48,55 +48,53 @@ initialState :: State
 initialState =
   { poller: pure unit
   , updateState: ChooseCreditCard
-  , chosenCreditCard: Nothing
   }
 
 component :: React.Component Props
-component = React.createComponent "update"
+component = React.createComponent "Update"
```
Comment (kaol) [inline packages/components/src/CreditCard/Update.purs:54]: It's an internal identifier, does capitalizing it affect anything?

Comment (ae-mo) [reply packages/components/src/CreditCard/Update.purs:54]: Not really, but I noticed that it was lower case while all other components have upper case :laughing:

**Review by kaol — APPROVED** [review]

**Pull request merged**